code/modules/admin/topic.dm

```diff
@@ -1743,6 +1743,33 @@
 		message_admins("[src.owner] replied to [key_name(H)]'s Syndicate message with: \"[input]\"")
 		to_chat(H, "You hear something crackle in your ears for a moment before a voice speaks.  \"Please stand by for a message from your benefactor.  Message as follows, agent. [input].  Message ends.\"")
 
+	else if(href_list["AdminFaxView"])
+		var/obj/item/fax = locate(href_list["AdminFaxView"])

```

Comment:
You need a better way of getting the fax than a locate like this. Fetch it from a known list instead.